stress/shell/src/main/java/alluxio/stress/cli/fuse/FuseIOBench.java

```diff
@@ -320,32 +375,38 @@ private void runInternal() throws Exception {
       mStartBarrierPassed = true;
 
       if (mParameters.mOperation == FuseIOOperation.LIST_FILE) {
```

Comment:
same try not to use big if else statement

Reply:
For this one I put the chunks of code into three separate functions and use a switch to call them, because even if we use the "if ... return; do something else" structure, it is still very large.